python/test_file_generator.py

```diff
@@ -1841,7 +1760,7 @@ def v674_1_file(self):
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.origination_fee = lar.origination_fee.map(lambda x: random.choice(("-1", "")))
+		lar.origination_fee = lar.origination_fee.map(lambda x: random.choice("HM"))
```

Comment:
Tried to change origination fee to an alphanumeric character to fail v674, but the edit produced formatting errors instead.